package.json

```diff
@@ -40,7 +40,8 @@
   "homepage": "https://github.com/vkbansal/react-contextmenu",
   "dependencies": {
     "classnames": "^2.2.5",
-    "object-assign": "^4.1.0"
+    "object-assign": "^4.1.0",
+    "prop-types": "^15.5.8"
```

Comment:
can you move `prop-types` to peerDependencies with `^15.0.0`? And also add it to devDependecies